<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCourse</class>
 <widget class="QDialog" name="AddCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Addbutton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>640</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>191</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Course Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Year</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Semester</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Course ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Coefficient</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>14</y>
     <width>361</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="NameLineedit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="yearedit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="semesteredit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="Idedit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="coefedit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="NumberofLectureedit">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>421</y>
     <width>361</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumberofLabedit">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>476</y>
     <width>361</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Numberoftutorialedit">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>531</y>
     <width>361</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>476</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lab</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="LabCheckBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>17</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="TutorialCheckBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>540</y>
     <width>17</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="LectureCheckBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>17</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>531</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tutorial</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>421</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lecture</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
